packages/neptune-extension/src/common/hooks/notebook.ts

```diff
@@ -1,37 +1,18 @@
-
 import React from 'react'
-import { useDispatch, useSelector } from 'react-redux'
+import { useDispatch } from 'react-redux'
 
 import { PlatformNotebook } from 'types/platform'
-
-import { setNotebook } from 'common/state/notebook/actions'
-
-import { getNotebook } from 'common/utils/upload'
+import { fetchNotebook } from "common/state/notebook/actions";
 
 export function loadInitialNotebook(platformNotebook: PlatformNotebook) {
 
   const dispatch = useDispatch()
 
   React.useEffect(() => {
-    async function fetchNotebook() {
-      const metadata = platformNotebook.getMetadata();
-
-      if (metadata.notebookId) {
-        // Load notebook from remote.
-        try {
-          const notebook = await getNotebook(metadata.notebookId)
-          dispatch(setNotebook(notebook))
-        } catch(e) {
-          // 404 or 403
-          dispatch(setNotebook(undefined))
-        }
-      }
```

Comment:
This case is a part of application bootstrap, you need to notify somehow that notebook id is not set and notebook will not be loaded. The display of buttons in App is delayed until this case resolves.